datasets/covtype.py

```diff
@@ -0,0 +1,26 @@
+from benchopt import BaseDataset, safe_import_context
+
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from copt.datasets import load_covtype
+
+
+class Dataset(BaseDataset):
+    name = "covtype"
+
+    install_cmd = 'conda'
+    requirements = ['pip:copt']
```

Comment:
I would have used the sklearn fetcher to avoid having to maintaining 2 fetchers

Reply:
I'll fix the madelon dataset to do the same with make_classification

Reply:
if you fetch with sklearn then the dep is wrong